### PR TITLE
NOISSUE - Golang and protoc version update

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Adds to the documentation the new updated versions of golang and protoc

### Which issue(s) does this PR fix/relate to?
It doesn't relate to previous issues

### List any changes that modify/break current functionality
No changes

### Have you included tests for your changes?
No tests

### Did you document any new/modified functionality?
Yes

### CODE REVIEW

File: docs/dev-guide.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ cd $SOMEPATH/mainflux
 
 ### Prerequisites
 
-Make sure that you have [Protocol Buffers](https://developers.google.com/protocol-buffers/) (version 3.11.4) compiler (`protoc`) installed.
+Make sure that you have [Protocol Buffers](https://developers.google.com/protocol-buffers/) (version 3.19.1) compiler (`protoc`) installed. Most versions from 2.3.1 should not give any problems, but 2.6.1, 3.0.2 and 3.14.0 are continuously tested.
```
Comment (drasko) [inline docs/dev-guide.md:18]: I think we can drop this last sentence - we should focus only on the last stable.

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/dev-guide.md (hunk 1 above)
Comment (rodneyosodo) [reply docs/dev-guide.md:18]: Okay

**Comment by rodneyosodo** [comment]
I have changed to a different branch

The current pull request is over [here #115 ](https://github.com/mainflux/docs/pull/115)

**Comment by drasko** [comment]
OK, closing this one then, but please take care - we avoid to closing the PRs that can be fixed.

**Pull request closed without merging**